backend/config/config_schema.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+  "properties": {
+    "docker": {
+      "type": "object",
+      "properties": {
+        "registry": {
+          "type": "string"
+        }
+      }
+    },
+    "operators": {
+      "type": "object",
+      "deprecated": true,
+      "properties": {
+        "operator": {
+          "type": "string",
+          "enum": ["docker", "k8s"]
+        },
+        "docker": {
+          "type": "object",
+          "deprecated": true,
+          "properties": {
+            "mountVolume": { "type": "string" },
+            "portRange": { "type": "string" },
+            "containerHost": { "type": "string" }
+          }
+        },
```

Comment:
Not just deprecated, the feature is removed, so we should also remove it from the schema.

Reply:
Yes, it shoud be removed there as well. Do you know if there are ways to generate the `config_template` out of the schema (so that we don't have to keep to files up to date)?

Reply:
There should be tools for that, but I'm not sure if it's not better to adjust it manually. I mean, we don't really have configuration changes that often (and adding individual attributes to the schema should be easy too). Besides, when maintaining it manually, we can extend the schema from time to time and increase the expressiveness (e.g. I specified an enum for the `storageAccessMode` or the `capabilities`).